t/add.t

```diff
@@ -207,6 +207,33 @@ my @cases = (
             },
         },
     },
+    # check that Max-Age supercedes Expires and that Max-Age <= 0 forces
+    # expiration
+    {
+        label   => "max-age supercedes expires",
+        request => "http://example.com/",
+        cookies => [
+            "lang=en-us; Max-Age=100; Expires=Thu, 1 Jan 1970 00:00:00 GMT",
+            "SID=31d4d96e407aad42; Expires=Thu, 3 Jan 4841 00:00:00 GMT",
+            "SID=0000000000000000; Max-Age=0",
```

Comment:
Could you please add a valid expires here (maybe after Max-Age)?  I'd like to verify that max-age supersedes an expires in the same cookie.

Reply:
Latest push now tests three conditions:

1. `Max-Age` in future, `Expires` in past

2. `Max-Age` in past, `Expires` in future

3. `Max-Age` == 0, `Expires` in future

Only the first cookie should be saved. Happy to add any additional tests...I just can't see what they should be.